x-pack/plugins/beats/server/routes/api/register_enroll_beat_route.js

```diff
@@ -80,7 +80,7 @@ export function registerEnrollBeatRoute(server) {
       try {
         const enrollmentToken = request.headers['kbn-beats-enrollment-token'];
         const { token, expires_on: expiresOn } = await getEnrollmentToken(callWithInternalUser, enrollmentToken);
-        if (!token || token !== enrollmentToken) {
+        if (!token) {
```

Comment:
I removed the comparison here because it was unnecessary. The comparison was being done in ES when we tried to `get` a doc with `_id` = `enrollment_token:${enrollmentToken}` from ES.